config/profile/rhdh/operator/plugin-deps/orchestrator/sample-greetings.yaml

```diff
@@ -0,0 +1,130 @@
+apiVersion: v1
```

Comment:
this should be removed before merging the PR.
but we can explore the idea of having in-house workflows to be delivered by RHDH.